packages/editor/src/store/test/selectors.js

```diff
@@ -2555,6 +2555,24 @@ describe( 'selectors', () => {
 	describe( 'getSelectedBlockClientId', () => {
 		it( 'should return null if no block is selected', () => {
 			const state = {
+				currentPost: {},
+				editor: {
+					present: {
```

Comment:
Can we extract this state and share between tests to avoid code duplication?

It would be also nice to wrap state with `deepFreeze` call to ensure it never mutates.

Reply:
The new variant doesn't have that much state in the tests.